server/game/cards/characters/07/grizzledminer.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../../drawcard.js');
+
+class GrizzledMiner extends DrawCard {
+
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            location: 'any',
+            targetType: 'player',
+            targetController: 'current',
+            effect: ability.effects.reduceSelfCost('marshal', this.getDiscount())
```

Comment:
Should be `() => this.getDiscount()` for dynamic costs. This will set the cost at the time of creating the card (which would be 0 if it weren't crashing)